tests/TestCase/Network/Http/Adapter/StreamTest.php

```diff
@@ -30,6 +104,13 @@ public function setUp()
             'Cake\Network\Http\Adapter\Stream',
             ['_send']
         );
+        stream_wrapper_register('cakephp', __NAMESPACE__ . '\CakeStreamWrapper');
```

Comment:
This is my idea :sunglasses: This change would allow to cover all part of the `_send` method.